.github/workflows/workflow_call.unit-test.yml

```
@@ -56,7 +55,16 @@ jobs:
         with:
           go-version: ${{ matrix.go-version }}
 
-      - name: unit tests
+      - name: Install GNU make on macOS
+        if: runner.os == 'macOS'
+        run: brew install make
```

More GNU utils are necessary to install on macOS.

```suggestion
        run: |
          brew install \
            make \
            grep \
            gawk \
            coreutils
```